### ISSUE TITLE
Broken stlite sharing examples

### ISSUE THREAD

**Issue by FdelMazo** [whitphx/stlite#675]
Both on brave and firefox I encountered an error on the live example site.

The [hello](https://edit.share.stlite.net/?sampleAppId=streamlit_hello) example works correctly

The [realtime image processing](https://edit.share.stlite.net/?sampleAppId=realtime_image_processing) fails with a micropip error

![image](https://github.com/whitphx/stlite/assets/25667102/22bf9e87-1345-4360-8d9c-dfa774a48eb0)

```js
Uncaught (in promise) PythonError: Traceback (most recent call last):
  File "/lib/python3.11/site-packages/micropip/_micropip.py", line 580, in install
    raise ValueError(
ValueError: Can't find a pure Python 3 wheel for: 'tornado<7,>=6.0.3', 'watchdog>=2.1.5; platform_system != "Darwin"', 'pyarrow>=6.0'
See: https://pyodide.org/en/stable/usage/faq.html#micropip-can-t-find-a-pure-python-wheel


    at new_error (pyodide.asm.js:9:14992)
    at pyodide.asm.wasm:0x152d67
    at pyodide.asm.wasm:0x159d1c
    at _PyCFunctionWithKeywords_TrampolineCall (pyodide.asm.js:9:126781)
    at pyodide.asm.wasm:0x19f6c6
    at pyodide.asm.wasm:0x28650b
    at pyodide.asm.wasm:0x1e0617
    at pyodide.asm.wasm:0x19fbad
    at pyodide.asm.wasm:0x19fe6e
    at pyodide.asm.wasm:0x19ff10
    at pyodide.asm.wasm:0x264a10
    at pyodide.asm.wasm:0x26a9f5
    at pyodide.asm.wasm:0x1a0038
    at pyodide.asm.wasm:0x19fcc8
    at pyodide.asm.wasm:0x159532
    at Module.callPyObjectKwargs (pyodide.asm.js:9:75373)
    at Module.callPyObject (pyodide.asm.js:9:75818)
    at wrapper (pyodide.asm.js:9:57648)
```

Could it be the bump of v0.42.3 to v0.42.4 or is the example site forced to use 0.42.3?

**Comment by whitphx** [comment]
Thank you for reporting this.
This is a bug introduced in 0.42.4, or #664.
Will fix it.

**Comment by whitphx** [comment]
This bug will be fixed in #676 and the Realtime Image Processing demo would work with it.
The Custom Components demo is still not working but it's because the latest version of `streamlit-cropper` updated its `streamlit` version requirement and it should be fixed when #671 is merged.
